### DISCUSSION TITLE
how do I import session from xshell?

### DISCUSSION THREAD

**Discussion by joshatt** [kingToolbox/WindTerm#590] [Q&A]
Marvellous functions, excited to try. 
I have 3 sessions in xshell. I exported one of them, to a C:\session_xsh.xts, then tried to import in Windterm. Did not figure it out. And with some random try, it created 3 session copies in xshell, making 6.
Any idea?

**Comment by kingToolbox** [comment]
Sorry for the late reply.

You can try the [conversion tool](https://github.com/kingToolbox/WindTerm/discussions/416) provided by @gh419. Although I haven't used it, I believe it should be useful.
